### PR TITLE
Add CONTRIBUTING.md [ci skip]

### PR DESCRIPTION
**Pull request by maclover7**
cc @rafaelfranca

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+Contributing to ActiveRecord Session Store
```
Comment (sikachu) [inline CONTRIBUTING.md]: Active Record is always written with space in between them, unless you are referring to the module. (Same goes for other components as well)

Comment (maclover7) [inline CONTRIBUTING.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -86,3 +86,12 @@ MyLogger.send :include, ActiveRecord::SessionStore::Extension::LoggerSilencer
 
 This silencer is being used to silence the logger and not leaking private
 information into the log, and it is required for security reason.
+
+## Contributing to ActiveRecord Session Store
```
Comment (rafaelfranca) [inline README.md]: Missing space here too in Active Record.

**Comment by maclover7** [comment]
@rafaelfranca Resolved your comment

**Pull request merged**